lib/KeyboardAwareMixin.js

```diff
@@ -112,7 +114,8 @@ const KeyboardAwareMixin = {
   /**
    * @param extraHeight: takes an extra height in consideration.
    */
-  scrollToFocusedInput: function (reactNode: Object, extraHeight: number = this.props.extraHeight) {
+  scrollToFocusedInput: function (reactNode: Object, extraHeight: number) {
```

Comment:
So 0.40 does fix this, but I don't think the latest released version of react-native uses this version yet, so might cause problems when people upgrade.
Happy to submit an update to move to 0.40.0 if you'd like (reverting this line and the line below)